Source/Network/NetworkLayer.swift

```diff
@@ -70,9 +79,10 @@ struct NetworkLayer {
         }
 
         if let headers = router.headers {
+            print("\nHTTP headers:", headers)
```

Comment:
Se for manter o print, seria legal colocar dentro de um if debug
```suggestion
            #if DEBUG
            print("\nHTTP headers:", headers)
            #endif
```

Reply:
feito, removido